docs/how_to_guides/create_and_edit_expectations/how_to_create_expectations_between_tables.rst

```diff
@@ -0,0 +1,38 @@
+.. _tutorial_create_expectations_between_tables:
+
+Create Expectations Between Tables
+===================================
+
+This tutorial covers the workflow of creating and editing expectations that relate to data stored in different tables or datasets.
+
+Unfortunately, no Expectations natively support cross-table comparisons today (but we hope to in the future!). Consequently, there are two available paths:
+
+1. Use "Check Assets", where you create a new table that joins the two tables; or
+2. Use :ref:`Evaluation Parameters`_ to supply relevant metrics to expectations.
+
+Check Asset Pattern
+-------------------------
+
+
+See :ref:`how_to__use_check_assets` for more information on the check asset pattern.
+
+Use Evaluation Parameters
+-------------------------
+
+To compare two tables using evaluation parameters, you create expectations for each table, and reference a property from one validation result in the other. To use evaluation parameters, both assets need to be validated during the same run.
```

Comment:
Capitalize Expectations

Reply:
Capitalize Validation Results and link on first reference.

Reply:
Capitalize Evaluation Parameters and link on first reference.

Reply:
updated (ref to eval parameters already existed earlier in doc, so only changed capitalization)